WordPress/src/main/java/org/wordpress/android/ui/main/utils/SiteRecordUtil.kt

```diff
@@ -0,0 +1,70 @@
+package org.wordpress.android.ui.main.utils
+
+import org.wordpress.android.fluxc.model.SiteModel
+import org.wordpress.android.ui.main.SiteRecord
+import java.util.Locale
+
+object SiteRecordUtil {
```

Comment:
Most of the code is copied from **SitePickerAdapter**, it's easier to do things in Kotlin. This object will be removed after refactors are completed.